bms_health.py

```diff
@@ -0,0 +1,23 @@
+from bms_constants import bms_thresholds
+
+
+def collect_anomalies(anomaly, bms_parameter, bms_parameter_value, bms_parameter_range):
+    if bms_parameter_value < bms_parameter_range['min']:
+        anomaly.append([bms_parameter, 'UNDER LIMIT'])
+    elif bms_parameter_value > bms_parameter_range['max']:
+        anomaly.append([bms_parameter, 'OVER LIMIT'])
+
+
+def bms_health_is_ok(status_report):

```

Comment:
The function `bms_health_is_ok` returns the parameters that are 'not' ok. Think of a better name for this.